components/app/admin/ReviewItem.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <div class="bg-base-200 rounded p-4 flex items-center">
+    <div class="flex-grow">
+      <!-- collapsed content -->
+      <div class="flex items-center space-x-2">
+        <app-logo :project="item.project" :size="32" />
+        <span class="text-primary font-bold">
+          {{ item.project.title }}
+        </span>
+        <span>•</span>
+        <span class="text-primary">
+          {{ $dateFns.formatDistanceToNowStrict(item.timestamp) }}
+        </span>
+        <span>•</span>
+        <span>
+          {{ categoryMap[item.project.category] }}
+        </span>
+      </div>
+
+      <!-- expanded content -->
+      <div v-if="expanded">
+        TODO: content
```

Comment:
- [ ] question(blocking): was this left intentionally?

Reply:
This was part of the initial work on the admin panel. Will be continued in: #818 
Added a commit to this branch to disable the /admin page for now.